message/models.py

```diff
@@ -1,11 +1,36 @@
 from django.db import models
 from player.models import Player
 from game_event.models import GameEvent
+from game_event_player.models import GameEventPlayer
 from django.core.validators import MinLengthValidator
+from django.core.exceptions import ValidationError
 
 
 class Message(models.Model):
     user_id = models.ForeignKey(Player, on_delete=models.CASCADE)
     game_event_id = models.ForeignKey(GameEvent, on_delete=models.CASCADE)
     time_sent = models.DateTimeField(auto_now_add=True)
     text = models.CharField(max_length=255, validators=[MinLengthValidator(1)])
+
+    @staticmethod
+    def create(player, game_event, text):
+        Message.validate_message(player, game_event, text)
+        message = Message(user_id=player,
+                          game_event_id=game_event,
+                          text=text)
+
+        message.save()
+        return message
+
+    @staticmethod
+    def validate_message(player, game_event, text):
```

Comment:
We should have unit tests that tests this method directly. We give different combinations of inputs and assert on the output (None) or the content of the error message.

Reply:
> We should have unit tests that tests this method directly. We give different combinations of inputs and assert on the output (None) or the content of the error message.

I added 3 tests:
 1. checks the validation is success when the input are correct
 2. checks the validation will raise 'ValidationError' when player that isn't on the event sends message,
 and checks the error message is correct
3. checks the validation will raise 'ValidationError' when player sends empty message/message that contains only space character, and checks the error message is correct

Reply:
Nice!